installing/installing_bare_metal_ipi/ipi-install-expanding-the-cluster.adoc

```diff
@@ -19,6 +19,7 @@ include::modules/ipi-install-preparing-the-bare-metal-node.adoc[leveloffset=+1]
 .Additional resources
 
 * xref:../../installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc#configuring-host-network-interfaces-in-the-install-config-yaml-file_ipi-install-installation-workflow[(Optional) Configuring host network interfaces in the install-config.yaml file]
```

Comment:
```suggestion
* xref:../../installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc#configuring-host-network-interfaces-in-the-install-config-yaml-file_ipi-install-installation-workflow[Optional: Configuring host network interfaces in the install-config.yaml file]
```

Reply:
I'd like skip this one for now since it requires me to change another module's heading, which lies under a chapter with such similar headings. Furthermore, this module has another instance of the usage of "(Optional)" [here](https://github.com/openshift/openshift-docs/pull/45918/files#diff-f90b46cde12693fcb811ca202b3759cea7652922c2b7f92da22716e5e8aaa180R14). It makes sense to change all of them or none of them.

Reply:
@abhatt-rh I think you can skip that for now. There are quite a few other titles with (Optional) rather than Optional: I can make those changes in a separate PR so there is consistency across the IPI doc set.